### DISCUSSION TITLE
DongshanPI-Seven

### DISCUSSION THREAD

**Discussion by DongshanPI** [DongshanPI/Dongshanpi-Docs#3] [General]
東山Pi柒号开发板-在线文档讨论交流区

**Comment by codebug8** [comment]
已知学员建议
* 驱动和实战应用
* A核和M核通讯及驱动
* 图像相关，从摄像头驱动到OpenCV或者部署一些简单的ai，主要不知道这块板可以做哪些相关的
* 点灯
* HelloWord
* linux的相关知识体系吧
* 感觉比较适合工控类应用，A核显示交互，或视觉识别，m核控制驱动
* 没有学过linux的东西，有单片机基础，在相信老师的教学影响力外加价格因素下买的。对于这款开发板来说都不知道他可以用来干嘛。希望可以通过这款开发板入门嵌入式。
* 想了解A和M直接怎么通讯的，两个核之间怎么分配IO资源的
* 智能仪表
* 还有就是能不能有以项目学习的。  不是单模块的模式。
* 音视频流处理
* 购买这款板子主要想学习双核异构，和linux应用开发。能不能搞一个专门网络通信的项目，或者通讯类的应用项目，因为目前来说都是物联网趋势，万物互联是大势所趋。比如所有通信的外设可以任意进行数据转发和处理，接受收据后可以进行分发给需要的应用层。
* 个人认为这块板子适合学习一些底层相关的东西，比如说适配uboot、kernal之类的
* 针对不同的板子硬件配置，学会bootroot，得到image，学会写特殊硬件的驱动，后面就是具体的各行各业的应用程序开发了
* 我个人来说,周边用到linux的机会不是很多，但是我想把公司的产品更深入开发些，想准备用linux，目的是各环节都要学习入门，最终能实现产品化。还是偏向驱动和应用层的东西。
* 我个人觉得不需要太偏底层的，比如用到I2C协议，直接怎么用，不必细讲I2C怎么通讯
* 想移植歌mtqq服务端作为我毕设智能家居的服务器

**Reply by codebug8** [reply]
最初的板子，关键如何让它从uboot到点亮小灯这个过程

**Comment by bosskingjin** [comment]
自我介绍一下：我搞单片机快20年了，像有些操作系统也用到产品中去，一直未能接触linux.我的经验是学习最快的时候，是有项目驱使你做的时候，那时候动力最足，但是对开发板来说不容易。平时我教别人的时候，就让他用公司的产品或者市面上的产品来学习。
我个人建议学习linux，可以逐步项目化，或者老师根据这个板子特点先定一个项目规划，然后一步步落实。主要多头到尾整个流程，就不用讲太细，毕竟学习的时间有限，个人要深入动手才行。
